### PR TITLE
[WIP] theme fix

### PR DESCRIPTION
**Pull request by DoParkEQ**

### CODE REVIEW

File: src/themes/typography.js [hunk 1]
```diff
@@ -1,52 +1,39 @@
-//The base pixel size is 8px.
 const typography = {
-  fontFamily: "Noto Sans",
+  fontFamily: 'Poppins',
```
Comment (YizhiCatherineZhang) [inline src/themes/typography.js:2]: Do we decided to change our font family?

Comment (DoParkEQ) [reply src/themes/typography.js:2]: No it is not supposed to be Poppins, but I am changing it to Open Sans

**Comment by SamSverko** [comment]
Is this PR still needed, @DoParkEQ and @YizhiCatherineZhang ? The Typography was addressed and updated in PR #80 .

**Comment by DoParkEQ** [comment]
It is no longer necessary as this is already addressed in another PR

**Pull request closed without merging**